scripts/gcloud.sh

```diff
@@ -165,117 +163,75 @@ gcloud_create_instance_template_if_not_exists() {
-  host_path="/var/hoprd"
-
-  log "checking for instance template ${name}"
-  if gcloud compute instance-templates describe "${name}" --quiet >/dev/null; then
-    log "instance template ${name} already present"
-
-    if [ "${skip_update_if_exists}" = "true" ]; then
-      # short-circuit, stop operation
-      return
-    fi
-
-    gcloud_delete_instance_template "${name}"
-  fi
+  echo ${metadata_value}
 
   log "creating instance template ${name}"
-
-  if [ "${no_args}" = "true" ]; then
-    eval gcloud compute instance-templates create-with-container "${name}" \
+  eval gcloud compute instance-templates create "${name}" \
       --machine-type=c2d-highcpu-2 \
-      --metadata=google-logging-enabled=true,google-monitoring-enabled=true,enable-oslogin=true \
       --maintenance-policy=MIGRATE \
       --tags=hopr-node,web-client,rest-client,portainer,healthcheck \
       --boot-disk-device-name=boot-disk \
       --boot-disk-size=20GB \
       --boot-disk-type=pd-balanced \
-      --image-family=cos-stable \
-      --image-project=cos-cloud \
-      --container-image="${image}" \
-      --container-env=^,@^DEBUG=hopr\*,@NODE_OPTIONS=--max-old-space-size=4096,@GCLOUD=1 \
-      --container-mount-host-path=mount-path="${mount_path}",host-path="${host_path}" \
-      --container-mount-host-path=mount-path=/var/run/docker.sock,host-path=/var/run/docker.sock \
-      --container-restart-policy=on-failure \
-      ${args} \
-      ${extra_args}
-  else
-    eval gcloud compute instance-templates create-with-container "${name}" \
-      --machine-type=c2d-highcpu-2 \
-      --metadata=google-logging-enabled=true,google-monitoring-enabled=true,enable-oslogin=true \
+      --image-family=debian-11 \
+      --image-project=hoprassociation \
       --maintenance-policy=MIGRATE \
-      --tags=hopr-node,web-client,rest-client,portainer,healthcheck \
-      --boot-disk-device-name=boot-disk \
-      --boot-disk-size=20GB \
-      --boot-disk-type=pd-balanced \
-      --image-family=cos-stable \
-      --image-project=cos-cloud \
-      --container-image="${image}" \
-      --container-env=^,@^DEBUG=hopr\*,@NODE_OPTIONS=--max-old-space-size=4096,@GCLOUD=1 \
-      --container-mount-host-path=mount-path="${mount_path}",host-path="${host_path}" \
-      --container-mount-host-path=mount-path=/var/run/docker.sock,host-path=/var/run/docker.sock \
-      --container-restart-policy=on-failure \
-      --container-arg="--admin" \
-      --container-arg="--adminHost" --container-arg="0.0.0.0" \
-      --container-arg="--healthCheck" \
-      --container-arg="--healthCheckHost" --container-arg="0.0.0.0" \
-      --container-arg="--identity" --container-arg="${mount_path}/.hopr-identity" \
-      --container-arg="--data" --container-arg="${mount_path}" \
-      --container-arg="--init" \
-      --container-arg="--api" \
-      --container-arg="--apiHost" --container-arg="0.0.0.0" \
-      ${args} \
-      ${extra_args}
-  fi
+      --metadata=${metadata_value}
```

Comment:
I know that we do not use it everywhere in the scripts, but please prefer double-quoting the string variables to prevent splitting.

Reply:
Solved